src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -58,25 +61,70 @@
 	private static final String PUT = "PUT";
 	private String canvasToken;
 	private String canvasURL;
+	
+	private String callType;
+	
+	private String ltiUrl;
+	private String ltiKey;
+	private String ltiSecret;
+	
 	ResourceBundle props = ResourceBundle.getBundle("sectiontool");
 
-	
 	public void init() throws ServletException {
 		M_log.debug(" Servlet init(): Called");
 	}
 	
-	protected void doGet(HttpServletRequest request,HttpServletResponse response){
+	public void fillContext(Context context, HttpServletRequest request) {
+		ViewToolContext vtc = (ViewToolContext)context;
+		vtc.put("variable", "happiness");
```

Comment:
Yes, this was test code to demo that I could actually put a variable called "variable" and swap it with text like "happiness". I will remove it. However, the method fillContext must remain because fillContext is always called by a "do" method in the velocityViewServlet class.